src/main/java/com/synopsys/integration/alert/provider/blackduck/descriptor/BlackDuckDistributionUIConfig.java

```diff
@@ -31,20 +31,16 @@
 import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.SelectConfigField;
 import com.synopsys.integration.alert.common.descriptor.config.field.TextInputConfigField;
-import com.synopsys.integration.alert.common.descriptor.config.ui.DescriptorMetadata;
 import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
-import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
-import com.synopsys.integration.alert.common.enumeration.DescriptorType;
-import com.synopsys.integration.alert.provider.blackduck.BlackDuckProvider;
 
 @Component
 public class BlackDuckDistributionUIConfig extends UIConfig {
 
-    @Override
-    public DescriptorMetadata generateDescriptorMetadata() {
-        return new DescriptorMetadata("Black Duck", "blackduck", BlackDuckProvider.COMPONENT_NAME, DescriptorType.PROVIDER, ConfigContextEnum.DISTRIBUTION, "laptop", createFields());
+    public BlackDuckDistributionUIConfig() {
+        super("Black Duck", "blackduck", "laptop");
```

Comment:
Encapsulate lowercase blackduck and laptop strings into static final strings in the descriptor class since they are the URL and font-awesome icon name.